tests/specs/check/check_workspace/main.ts

```diff
@@ -0,0 +1,8 @@
+// We shouldn't get diagnostics from this import under this check scope.
```

Comment:
Unlike the LSP, we actually do want to see diagnostics under different scopes because they might contribute to errors seen in the current scope. I've updated the next PR to handle this and group/display the diagnostics by config.